PagerDuty/public-support-scripts
# T2D2-51 general readme cleanup

**Link back to Jira ticket:** https://pagerduty.atlassian.net/browse/T2D2-51

## Description

Update general readme as per T2D2-51.

## Implementation

Mentions of externally hosted files removed, list of scripts included in the repo updated.

### Steps to test changes
* Make sure readme does not mention external scripts
* Make sure list of scripts included in the readme reflects the current state of the repo.

## Documentation 
- [] There is documentation that needs to be updated upon merging these changes

Links to any documentation to be updated:
